build/repo.targets

```diff
@@ -196,27 +196,45 @@
       XmlInputPath="$(RepositoryFileWithCommit)"
       Query="//Repository[@Include='%(Repository.Identity)']/@Commit"
       Value="%(Repository.Commit)" />
+
+    <XmlPoke
```

Comment:
Starting to think xmlpoke isn't the right solution. This is fine for now, but maybe a RepoTasks project would simplify this.